scripts/symlink_c_libs_without_fingerprint.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -e
+
+function usage() {
+    echo "$0: symlink C libs to names without the variable fingerprint part"
```

Comment:
What is the variable fingerprint part?

Reply:
like in the target/deps dir we get libthfe_c_api_dynamic_buffer-somehash.a so if we don't symlink to a "constant" name our builds would be a nightmare to make work, so I look for the lib that has the same name and is the most recent and say it's the lib we want